pom.xml

```diff
@@ -53,6 +55,11 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-actuator</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20230227</version>
```

Comment:
We normally define properties for the version numbers. Not sure what that adds, but consistent at least.

Reply:
pushed a new pom with the version number constants